OOVisualization/src/declarations/VClass.cpp

```diff
@@ -105,7 +105,10 @@ void VClass::initializeForms()
 				->put(1, 4, item<VList>(&I::declarations_,
 						[](I* v) {return v->node()->subDeclarations()->size() > 0 ? v->node()->subDeclarations() : nullptr;},
 						[](I* v){return &v->style()->declarations();}))
-				->put(1, 5, (new DynamicGridFormElement())->setSpacing(10, 10)->setMargins(10)
+				->put(1, 5, item<VList>(&I::metaCalls_,
```

Comment:
OK, I know I said that visualizations don't have be perfect now, but I think with just a little bit more effort we can make a big improvement. In *All* the places where you visualize metacalls (so Method, Class, Module, and Project):
* Always directly feed the list node, without checking for its size (e.g. as in `VMethod`)
* always specify a style. * Don't just borrow another style, but introduce a dedicated style for metaDeclarations
* This means you'll have to extend the corresponding VXXXStyle classes as well with the additional property. Watch out for old style implementations, which will require a change in both the .h and the .cpp file
* You'll also have to extend the corresponding XML style files. Use this style:
```xml    
    <metaCalls prototypes="item/VList/default">
		<shape prototypes="shape/Box/default">
			Box
			<cornerRadius>1</cornerRadius>
			<cornerType>0</cornerType>
			<backgroundBrush>
				<style>1</style>
				<color>#f0f0ff</color>
			</backgroundBrush>
			<outline>
				<width>1</width>
				<color>#8080ff</color>
			</outline>
		</shape>
		<itemsStyle prototypes="layout/SequentialLayout/default">
			<direction>1</direction>
			<spaceBetweenElements>10</spaceBetweenElements>
			<notLocationEquivalentCursors>true</notLocationEquivalentCursors>
		</itemsStyle>
		<selectedTip>
			<symbol>add meta calls</symbol>
		</selectedTip>
	</metaCalls>
```
Feel free to change the colors though :)

These changes should make for a very nice editing experience.